### PR TITLE
feat: rebrand website

### PR DESCRIPTION
**Pull request by isaackps**
## Summary

Rebranding entire website with new logo and colors

## Changes

- Update entire website color schemes
- Update with new Logo
- clean up css

## Issues

https://www.pivotaltracker.com/n/projects/2424361/stories/181982208

### CODE REVIEW

File: src/components/HomePageContent/WelcomeSection.tsx [hunk 1]
```diff
@@ -10,29 +10,29 @@ const DescriptionSection: FunctionComponent = () => {
   return (
     <section id="welcome" className="bg-cerulean-50 h-full text-gray-700 md:pt-16 mb-4">
       <div
-        className="relative bg-135% bg-right-bottom bg-no-repeat py-16 md:bg-auto md:h-full md:bg-right-top"
+        className="relative bg-[length:135%] bg-right-bottom bg-no-repeat py-16 md:bg-auto md:h-full md:bg-right-top"
         style={{ backgroundImage: "url('/static/images/home/welcome/map.png')" }}
       >
-        <div className="container md:h-140">
+        <div className="container md:h-[35rem]">
           <div className="md:w-6/12">
```
Comment (simboonlong) [inline src/components/HomePageContent/WelcomeSection.tsx:17]: sweet tw3

File: src/tailwind.js [hunk 2]
```diff
@@ -13,7 +13,10 @@ const localConfig = {
       "wave-lines-light": "url('/static/images/common/wave-lines-light.png')",
     },
     fontFamily: {
-      roboto: ["Roboto", "sans-serif"],
+      "gilroy-light": ["Gilroy-Light", "sans-serif"],
+      "gilroy-medium": ["Gilroy-Medium", "sans-serif"],
+      "gilroy-bold": ["Gilroy-Bold", "sans-serif"],
+      "gilroy-extrabold": ["Gilroy-ExtraBold", "sans-serif"],
       ubuntu: ["Ubuntu", "sans-serif"],
     },
```
Comment (simboonlong) [inline src/tailwind.js]: nitpick (non-blocking): these are inherited from [ui components](https://github.com/TradeTrust/tradetrust-ui-components/blob/master/src/tailwind.js#L14). if im not wrong, will work without

**Comment by simboonlong** [comment]
thanks for all the images replacement, do checkout wats wrong with ci

File: src/tailwind.js (hunk 2 above)
Comment (isaackps) [reply src/tailwind.js]: ok will update it to be like the one in creator.

**Review by simboonlong — APPROVED** [review]

**Pull request merged**